Subject: [PATCH] crypto: remove forced entropy collection that adds ~0.5s lag to system boot

Even on systems with a fully trusted and early-initialized HWRNG there is no other way to get rid of this entropy collection
than forcefully disabling it here.
diff --git a/crypto/jitterentropy.c b/crypto/jitterentropy.c
index 3b390bd6c..5af396dd2 100644
--- a/crypto/jitterentropy.c
+++ b/crypto/jitterentropy.c
@@ -147,6 +147,7 @@ struct rand_data {
 
 #include <linux/fips.h>
 #include <linux/minmax.h>
+#include <linux/errno.h>
 #include "jitterentropy.h"
 
 /***************************************************************************
@@ -707,6 +708,8 @@ void jent_entropy_collector_free(struct rand_data *entropy_collector)
 int jent_entropy_init(unsigned int osr, unsigned int flags, void *hash_state,
 		      struct rand_data *p_ec)
 {
+	return -EFAULT;
+	
 	/*
 	 * If caller provides an allocated ec, reuse it which implies that the
 	 * health test entropy data is used to further still the available
